/usr/bin/iceauth:  creating new authority file /run/user/1000/ICEauthority
gpg-agent[645]: WARNING: "--write-env-file" is an obsolete option - it has no effect
gpg-agent: a gpg-agent is already running - not starting a new one

(xfce4-session:595): xfce4-session-WARNING **: 13:50:49.509: gpg-agent returned no PID in the variables

(xfce4-session:595): xfce4-session-WARNING **: 13:50:49.513: xfsm_manager_load_session: Something wrong with /home/ctpc/.cache/sessions/xfce4-session-ctpc-dell:0, Does it exist? Permissions issue?

(xfwm4:648): xfwm4-WARNING **: 13:50:49.794: Unsupported GL renderer (llvmpipe (LLVM 8.0, 256 bits)).

(xfwm4:648): xfwm4-WARNING **: 13:50:49.795: Screen is missing required GL renderer, GL support disabled.

(wrapper-2.0:677): Gtk-WARNING **: 13:50:52.616: Negative content width -1 (allocation 1, extents 1x1) while allocating gadget (node button, owner GtkToggleButton)

(wrapper-2.0:680): Gtk-WARNING **: 13:50:52.663: Negative content width -1 (allocation 1, extents 1x1) while allocating gadget (node button, owner PulseaudioButton)

(xfce4-session:595): xfce4-session-WARNING **: 13:50:53.393: Unable to launch "/etc/skel/.config/autostart/xfce-pbw.sh" (specified by autostart/xfce-panel-workaround.desktop): Failed to execute child process “/etc/skel/.config/autostart/xfce-pbw.sh” (No such file or directory)
vmware-user: could not open /proc/fs/vmblock/dev

** (xfce4-clipman:698): WARNING **: 13:50:56.338: Unable to register GApplication: An object is already exported for the interface org.gtk.Application at /org/xfce/clipman

(xfce4-clipman:698): GLib-GIO-CRITICAL **: 13:50:56.339: g_application_get_is_remote: assertion 'application->priv->is_registered' failed

(xfce4-clipman:698): GLib-WARNING **: 13:50:56.339: g_set_application_name() called multiple times

(pamac-tray:705): Gdk-CRITICAL **: 13:50:57.201: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
blueman-applet 13.50.57 ERROR    PluginManager:90 load_plugin: Unable to load plugin module AppIndicator
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/blueman/plugins/applet/AppIndicator.py", line 7, in <module>
    require_version('AppIndicator3', '0.1')
  File "/usr/lib/python3.7/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace AppIndicator3 not available

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/blueman/main/PluginManager.py", line 88, in load_plugin
    importlib.import_module(self.module_path.__name__ + ".%s" % plugin)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/lib/python3.7/site-packages/blueman/plugins/applet/AppIndicator.py", line 9, in <module>
    raise ImportError("AppIndicator3 not found")
ImportError: AppIndicator3 not found
blueman-applet 13.50.57 ERROR    PluginManager:90 load_plugin: Unable to load plugin module KillSwitch
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/blueman/main/PluginManager.py", line 88, in load_plugin
    importlib.import_module(self.module_path.__name__ + ".%s" % plugin)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/lib/python3.7/site-packages/blueman/plugins/applet/KillSwitch.py", line 23, in <module>
    raise ImportError('Hardware kill switch not found')
ImportError: Hardware kill switch not found
blueman-applet 13.50.58 WARNING  PluginManager:147 __load_plugin: Not loading PPPSupport because its conflict has higher priority
blueman-applet 13.50.58 WARNING  PluginManager:147 __load_plugin: Not loading DhcpClient because its conflict has higher priority

(wrapper-2.0:677): Gdk-CRITICAL **: 13:51:01.462: Window 0x559e31b35000 has not been made visible in GdkSeatGrabPrepareFunc
blueman-applet 13.51.15 WARNING  DiscvManager:109 update_menuitems: warning: Adapter is None
thunar-volman: Unsupported USB device type "usb".

(wrapper-2.0:677): Gdk-CRITICAL **: 14:03:24.924: Window 0x559e31b35000 has not been made visible in GdkSeatGrabPrepareFunc

(wrapper-2.0:677): Gdk-CRITICAL **: 15:16:51.379: Window 0x559e31b35000 has not been made visible in GdkSeatGrabPrepareFunc
